docs/pages/base/api/use-slot-props.json

```diff
@@ -0,0 +1,7 @@
+{
+  "inputParams": {},
+  "returnValue": {},
+  "name": "useSlotProps",
+  "filename": "/packages/mui-base/src/utils/useSlotProps.ts",
+  "demos": "<ul></ul>"
+}
```

Comment:
Should this be excluded from the public? cc @michaldudak

Reply:
I made the script to not throw if there are no demos, but we will likely need to do another pass and check each of the docs pages.

Reply:
It could be useful for building custom libraries, so let's leave it in.

Reply:
Removing it from now, until we have demos for the hook as decided on the backlog grooming. We will need to anyway review the input params descriptions.